server/app/app.go

```diff
@@ -107,25 +108,25 @@ func NewPlasmaMVPChain(logger log.Logger, db dbm.DB, traceStore io.Writer, optio
 		app.feeAmount = app.feeAmount.Add(app.feeAmount, amt)
 		return nil
 	}
-	app.Router().AddRoute(msgs.SpendMsgRoute, handlers.NewSpendHandler(app.utxoStore, app.plasmaStore, nextTxIndex, feeUpdater))
-	app.Router().AddRoute(msgs.IncludeDepositMsgRoute, handlers.NewDepositHandler(app.utxoStore, app.plasmaStore, nextTxIndex, plasmaClient))
+	app.Router().AddRoute(msgs.SpendMsgRoute, handlers.NewSpendHandler(app.outputStore, app.blockStore, nextTxIndex, feeUpdater))
+	app.Router().AddRoute(msgs.IncludeDepositMsgRoute, handlers.NewDepositHandler(app.outputStore, app.blockStore, nextTxIndex, plasmaClient))
 
 	// custom queriers
 	app.QueryRouter().
-		AddRoute("utxo", store.NewUtxoQuerier(utxoStore)).
-		AddRoute("plasma", store.NewPlasmaQuerier(plasmaStore))
+		AddRoute("tx", store.NewOutputQuerier(outputStore)).
```

Comment:
change to `"output"`

Reply:
actually we should just add public consts for these strings, OutputRoute or something like that